.ignore

```diff
@@ -0,0 +1 @@
+ports/rust
```

Comment:
see other comment about moving this to `/rust`

Reply:
I still clone it to `ports/rust` rather than `rust` in the Makefile.